lib/item_stream_handler.rb

```diff
@@ -64,10 +71,13 @@ def handle (event)
         # Presence of 'duedate' indicates it's checked-out
         if decoded && decoded['status'] && ! decoded['status']['duedate'].nil?
           checkout = Checkout.from_item_record decoded
-          add_checkout checkout
-
-          checkout_count += 1
-          update_count checkout
+          unless RECENT_IDS[checkout.id] && RECENT_IDS[checkout.id] - Time.now < ENV[CHECKOUT_ID_EXPIRE_TIME].to_i
```

Comment:
Could we have a test around this functionality? And handle nil ENV var?

Reply:
I've added some tests. Also, I've made sure to call `.to_i` on all the ENV["CHECKOUT_ID_EXPIRE_TIMES"]'s, so if this is nil it should be equivalent to no deduping. Let me know if you think there is a better default.